src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -497,6 +497,54 @@ BOOST_AUTO_TEST_CASE(SELUFunctionDerivativeTest)
       selu.Lambda() * selu.Alpha() - arma::mean(activations))), 10e-4);
 }
 
+/*
+ * Implementation of the CELU activation function test. The function is
+ * implemented as CELU layer in the file celu.hpp
+ *
+ * @param input Input data used for evaluating the CELU activation function.
+ * @param target Target data used to evaluate the CELU activation.
+ */
+void CheckCELUActivationCorrect(const arma::colvec input,
+                                     const arma::colvec target)
```

Comment:
Hi @gaurav-singh1998, Could also please fix the indentation here. The `c`'s should align. Thanks.